Makefile

```diff
@@ -9,6 +9,7 @@ export TEST_OS
 TARFILE=$(RPM_NAME)-$(VERSION).tar.xz
 NODE_CACHE=$(RPM_NAME)-node-$(VERSION).tar.xz
 SPEC=$(RPM_NAME).spec
+PREFIX ?= /usr/local
```

Comment:
I think `=` is fine here, actually.  You can still override it on the commandline of `make`.

Reply:
Hmm, it's an "external" variable, unlike the others which are internal and not meant to be overridden. But I suppose that's more of a visual distinction. "not from command line" is `override` I suppose, which is rarely used.

I temporarily hanged it to `=`, and it works. I suppose we should clean this up more thoroughly in starter-kit, probably with rearranging and commenting, so I changed it back for this PR (which is supposed to be a cherry-pick).